pkg/api/api_test.go

```diff
@@ -93,6 +93,7 @@ type testServerOptions struct {
 	PostageContract    postagecontract.Interface
 	Post               postage.Service
 	Steward            steward.Interface
+	BatchSyncStatus    postage.SyncStatus
```

Comment:
not sure whether an fully fledged interface is needed here. perhaps a pluggable function would do